### PR TITLE
Hchow/tlt2650 permissions tests for class roster tool

### PR DESCRIPTION
**Pull request by hktest**
-  update of existing permissions tests for class roster tool.
- updated test site and updated test data to reflect the ACP criteria in TLT-2650.  

Test works.  There are some roles that are failing due to migrations.  Will run regression suite once all tests are checked in.

### CODE REVIEW

File: selenium_tests/class_roster/class_roster_permission_tests.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def test_roles_access(self, user_id, given_access):
                 self.course_admin_dashboard_page
                     .class_roster_button_is_displayed(),
                 'User {} should see the class roster button on page but '
```
Comment (elliottyates) [inline selenium_tests/class_roster/class_roster_permission_tests.py]: I think you need to update this string as well to include the role (see e.g. https://github.com/Harvard-University-iCommons/canvas_manage_course/pull/18/files#diff-1f4bb4d36f243029f9a55633ae7b615dR41).

**Comment by hktest** [comment]
Closing this PR for now.

File: selenium_tests/class_roster/class_roster_permission_tests.py [hunk 2]
```diff
@@ -54,5 +54,5 @@ def test_roles_access(self, user_id, given_access):
         else:
             raise ValueError(
                 'given_access column for user {} must be either "yes" or '
```
Comment (elliottyates) [inline selenium_tests/class_roster/class_roster_permission_tests.py]: ditto above

File: selenium_tests/class_roster/class_roster_permission_tests.py (hunk 1 above)
Comment (hktest) [inline selenium_tests/class_roster/class_roster_permission_tests.py]: Great catch.  Somehow I missed this - thanks. Will have it fixed for the new PR.

**Pull request closed without merging**